### ISSUE TITLE
SC 3.3.8 Redundant entry -- problem scenario is not inherently blocking, should be AA or AAA

### ISSUE THREAD

**Issue by wittjeff** [w3c/wcag#1446]
on behalf of edX

We recognize that redundant entry is a negative-usability scenario and that such scenarios can cumulatively be de facto blocking accessibility issues.  Still, a given redundant entry scenario is not necessarily blocking accessibility. We believe A-level Success Criteria should be reserved for scenarios that are inherently blocking for some users in all cases. Therefore we recommend that this SC be given AA or AAA status.

**Comment by Helixopp** [comment]
Having short term memory impairments makes this issue a Level A accessibility barrier. Regardless of how much time passes between the time a user initially receives or provides information that must be provided again, short term memory loss can , and often does, make that man impossibility. As a survivor of a traumatic brain injury, I speak with personal experience, in addition to the available research on the topic. Often times, users with COGA disabilities will end their session after not being able to provide the redundant info. After that they forget all about the session, and never return.

**Comment by Helixopp** [comment]
Draft Response: The barrier that this issue poses to individuals with short term memory impairments makes this issue a Level A accessibility barrier. Regardless of how much time passes between the time a user initially receives or provides information that must be provided again, short term memory loss can , and often does, make that an impossibility. Often times, users with learning and cognitive disabilities will end their session after not being able to provide the redundant information. Shortly after ending their session, it quickly fades from their memory and they never remember to return and make another attempt.

**Comment by alastc** [comment]
Draft response:
<hr>
You indicate in your question that EdX believes "A-level Success Criteria should be reserved for scenarios that are inherently blocking for some users in all cases." The Working Group has not clarified specific criteria to indicate which Level Success Criteria need to be a part of, and elected to place the Redundant Entry SC into Level A because it poses a significant barrier to individuals with short term memory impairments. 

Users with short term memory impairments are often unable to complete tasks due to the amount of information that needs to be entered, and this SC places limits on information that needs to be entered more than once that are readily addressed and help reduce the barrier for these users.

See this [wiki page of a discussion of the history during WCAG 2.0 - 2.2](https://www.w3.org/WAI/GL/wiki/WCAG_2.x_Priority_levels_discussion).
